_po/ja/how-to/laravel.po

```diff
@@ -193,38 +264,57 @@ msgid ""
 "            return static::query();\n"
 "        }"
 msgstr ""
+"    public static function search($keyword)\n"
+"    {\n"
+"        if(empty($keyword)){\n"
+"            return static::query();\n"
+"        }"
 
 msgid "        $search_columns = ['title', 'body'];"
-msgstr ""
+msgstr "        $search_columns = ['title', 'body'];"
 
 msgid "        $search_query = static::query();"
-msgstr ""
+msgstr "        $search_query = static::query();"
 
 msgid ""
 "        foreach($search_columns as $column){\n"
 "            $search_query->orWhereRaw(\"$column &@~ ?\", [$keyword]);\n"
 "        }"
 msgstr ""
+"        foreach($search_columns as $column){\n"
+"            $search_query->orWhereRaw(\"$column &@~ ?\", [$keyword]);\n"
+"        }"
 
 msgid ""
 "        return $search_query;\n"
 "    }\n"
 "}\n"
 "```"
 msgstr ""
+"        return $search_query;\n"
+"    }\n"
+"}\n"
+"```"
 
 msgid ""
 "#### Migration\n"
 "Now we create a database migration for our Post model `database/migrations/<ti"
 "mestamp>_create_posts_table.php`\n"
 "Here are its original state and after state we modified it:"
 msgstr ""
+"#### マイグレーション\n"
+"次にPostモデル`database/migrations/<timestamp>_create_posts_table.php`用の”
+”マイグレーションファイルを作成します\n"
+"こちらが変更前、変更後の内容となります:"
 
 msgid ""
 "use Illuminate\\Database\\Migrations\\Migration;\n"
 "use Illuminate\\Database\\Schema\\Blueprint;\n"
 "use Illuminate\\Support\\Facades\\Schema;"
 msgstr ""
+"use Illuminate\\Database\\Migrations\\Migration;\n"
+"use Illuminate\\Database\\Schema\\Blueprint;\n"
+"use Illuminate\\Support\\Facades\\Schema;"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.